src/language/utils.ts

```diff
@@ -76,4 +76,11 @@ module Lint {
             callback(scanner);
         }
     }
+
+      export function isBlockScopedVariable(node: ts.VariableDeclaration): boolean {
+          // determine if the appropriate bit in the parent (VariableDeclarationList) is set,
```

Comment:
If you're making this a Lint-wide utility function, you might want to check and see if the parent is a CatchClause, since this is a thing that can happen. I am not sure that counts as block-scoped though, depends on how you're using it.